pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/bestpractices/xml/SystemPrintln.xml

```diff
@@ -55,12 +50,6 @@ public class Foo {
     <test-code>
         <description>#1217 SystemPrintln always says "System.out.print is used"</description>
         <expected-problems>4</expected-problems>
-        <expected-messages>
```

Comment:
Since we have now anyway always the same message, this test case doesn't make sense anymore. Should we simply remove it?